crates/worker/src/partition/mod.rs

```diff
@@ -130,8 +130,7 @@ where
             last_applied_lsn.unwrap_or(Lsn::INVALID),
         );
 
-        let mut action_collector = ActionCollector::with_capacity(128);
-        let mut effects_buffer = Effects::with_capacity(128);
+        let mut action_collector = ActionCollector::with_capacity(32);
```

Comment:
Looks like a carefully chosen value 😄

Reply:
You are right. This value is as good as any other value so why put it there in the first place. Will remove it.